chp00_introduction/NOC_I_01_RandomWalkTraditional/index.html

```diff
@@ -1,5 +1,5 @@
 <head>
-  <script language="javascript" type="text/javascript" src="//cdnjs.cloudflare.com/ajax/libs/p5.js/0.5.7/p5.js"></script>
+  <script language="javascript" type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/p5.js/0.5.16/p5.js"></script>
```

Comment:
Thank you for this! I am under the impression that it's best to lave `https` out this way the get request for the library can adapt to `http` or `https` depending on where it is served. This does, however, break the examples if the user does not run a local server. What do you think?

Reply:
According to the response headers, with a `http` request, CDNJS performs an internal redirect to the `https` URL, whether the file is served locally (`file:///`) or remotely (`http://`). So `https` would spare one request.
When I downloaded the examples, the first thing I've done was to double-clicking an `index.html` file, hoping to see something actually moving across the screen. Without scheme in the URL there was just a disappointing blank window, it worked with a local server but that was not my first move. So I think that this is more « user friendly » to have a full URL and since CDNS ends up serving an `https` response, we might as well request for it in the first place.
I hope this answers your question (I’m not super comfortable with my english).